tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import random
+import string
+
+import pytest
+
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, check_time_travel, detect_initial_scan
+from wazuh_testing.tools import PREFIX
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import remove_file
+from wazuh_testing.tools.services import control_service
+
+
+# Marks
+
+
+pytestmark = [pytest.mark.tier(level=0)]
+
+# Variables
+
+
+test_directories = [os.path.join(PREFIX, 'testdir1')]
+
+directory_str = ','.join(test_directories)
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf.yaml')
+testdir1 = test_directories[0]
+
+# configurations
+
+
+conf_params = {'TEST_DIRECTORIES': directory_str, 'MODULE_NAME': __name__}
+
+parameters, metadata = generate_params(extra_params=conf_params)
+
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+# Fixtures
+
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='function')
+def create_and_restore_large_file(request):
+    if not os.path.exists(testdir1):
+        os.mkdir(testdir1)
+
+    file_size = 1024 * 1024 * 768   # 805 MB
+    chunksize = 1024 * 768
+    file_path = os.path.join(testdir1, 'large_file')
+
+    with open(file_path, "a") as f:
+        while os.stat(file_path).st_size < file_size:
+            f.write(random.choice(string.printable) * chunksize)
+    yield
+
+    remove_file(file_path)
+
+
+@pytest.fixture()
+def wait_for_initial_scan():
+    """Fixture that waits for the initial scan, independently of the configured mode."""
+    detect_initial_scan(wazuh_log_monitor)       
```

Comment:
There are other fixtures that do the same that this one. Maybe is a better idea to use `wait_for_fim_start`.